Makefile

```diff
@@ -773,3 +774,12 @@ check-example-config-doc: generate-example-config-doc
 		echo -e "(Don't forget to check in the generated files when finished)\n"; \
 		exit 1; \
 	fi
+
+k3d-loki:
```

Comment:
The name is a little confusing. How about `tanka-loki` instead?

Reply:
maybe `helm-loki-cluster`? it doesn't use `tanka` to install Loki, so I think `tanka-loki` is also misleading.

Reply:
or `k3d-loki-cluster`? It spins up a `k3d` cluster using `tanka`, which it then `helm installs` Loki into.

Reply:
How about `dev-k3d-...`